2-numpy/exercises.ipynb

```diff
@@ -920,7 +920,8 @@
     "    \n",
     "gt.show_in_notebook(display_length=10)\n",
     "    \n",
-    "gt.add_column((gt[\"vy\"] * gt[\"x\"] - gt[\"vx\"] * gt[\"y\"]), name=\"Lz\")    \n",
+    "gt.add_column((gt[\"vy\"] * gt[\"x\"] - gt[\"vx\"] * gt[\"y\"]), name=\"Lz\")\n",
+    "gt.add_column(np.hypot(gt[\"x\"], gt[\"y\"]), name=\"r\")\n",
```

Comment:
The code here isn't really using any noteworthy features of `QTable.add_column()`, so we might as well edit it some more:
```suggestion
    "gt[\"Lz\"] = gt[\"vy\"] * gt[\"x\"] - gt[\"vx\"] * gt[\"y\"])\n",
    "gt[\"r\"] = np.hypot(gt[\"x\"], gt[\"y\"])\n",
```

Reply:
I think it is worth using `QTable.add_column()` for two reasons;
1. It makes it very clear what the code is doing, and that a `Table` is not just a dictionary.
2. It informs students that the method exists by presenting an example.

Reply:
A `Table` _is_ a dictionary of columns with some additional convenience methods and the ability to store extra metadata, even if it isn't a subclass of `dict`. I wouldn't bother using `Table.add_columns()` unless I really cared about column ordering.

Reply:
Fair point, I accept.